integration-tests/bats/schema-changes.bats

```diff
@@ -37,6 +37,16 @@ teardown() {
     [[ ! "$ouput" =~ "Merge failed" ]] || false
 }
 
+@test "schema-changes: dolt schema alter column preserves table checks" {
+    run dolt sql -q "alter table test add constraint test_check CHECK (c2 < 12345);"
+    [ "$status" -eq 0 ]
```

Comment:
So if all you want to do is check that the command returns 0 just omit `run`. For example 41 and 42 can just be:
`dolt sql -q "alter table test add constraint test_check CHECK (c2 < 12345);"`

`run` populates $status, $output, @lines, etc